src/index.ts

```diff
@@ -347,7 +350,7 @@ function tokenBase(stream: StringStream, state: State) {
       keywordsMoreBlockAndTransactionProperties,
       cur,
     ) ||
-    (cur in keywordsBlockAndTransactionProperties &&
+    (keywordsBlockAndTransactionPropertiesKeys.indexOf(cur) !== -1 &&
       ((keywordsBlockAndTransactionProperties as any)[cur] as Array<
```

Comment:
I just double checked and in a version vendored locally we have it written like this:
```typescript
  if (
    Object.prototype.propertyIsEnumerable.call(
      keywordsMoreBlockAndTransactionProperties,
      cur,
    ) ||
    (Object.prototype.hasOwnProperty.call(
      keywordsBlockAndTransactionProperties,
      cur,
    ) &&
      keywordsBlockAndTransactionProperties[
        cur as keyof typeof keywordsBlockAndTransactionProperties
      ].some(function (item) {
        return stream.match(`.${item}`);
      }))
  ) {
    return 'variable-2';
  }
```

no strong preference but it achieves the same effect without having to iterate through the keys. wdyt?

Reply:
Fine either way!

Reply:
Though I'm curious why check both membership methods

Reply:
oh yeah you certainly don't need both haha want to make that change? can merge this PR in after